### PR TITLE
Add django-simple-history to keep track of Reimbursement changes

### PR DESCRIPTION
**Pull request by cuducos**
As suggested in #174 (and added Django Admin — for admins only, by now — to better visualize the new feature)

image

### CODE REVIEW

File: jarbas/core/admin.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        'state',
+    )
+
+    def is_suspicious(self, obj):
+        return obj.suspicions is not None
+
+    is_suspicious.short_description = 'Suspicious'
+    is_suspicious.boolean = True
+
+
```
Comment (jtemporal) [inline jarbas/core/admin.py]: I thought you were going to show the historical records in the admin. And if you weren't why not?

Comment (cuducos) [inline jarbas/core/admin.py]: We are:

Comment (jtemporal) [inline jarbas/core/admin.py]: Oooh I see it now, damn top right button!

**Comment by cuducos** [comment]
> Also I would add to README the ref to Django Docs were it shows the `createsuperuser` command

Good idea! Done in 42d29c7

**Pull request merged**